### ISSUE TITLE
pyexasol 0.18.1 is incompatible with websocket-client == 1.0.0

### ISSUE THREAD

**Issue by smaspe** [badoo/pyexasol#75]
Hi there,

When using this particular combination of dependencies:

```
pyexasol = 0.18.1
websocket-client = 1.0.0
```

Using a context manager:

```
with pyexasol.connect(...) as c:
    st = c.execute("select 1")
    st.fetchall()
```

I see this error:

```
  File "/***/virtualenvs/***-njgrPu0H/lib/python3.7/site-packages/pyexasol/connection.py", line 836, in __exit__
    self.close()
  File "/***/virtualenvs/***-njgrPu0H/lib/python3.7/site-packages/pyexasol/connection.py", line 479, in close
    self._ws.close()
  File "/***/virtualenvs/***-njgrPu0H/lib/python3.7/site-packages/websocket/_core.py", line 491, in close
    self.sock.settimeout(sock_timeout)
AttributeError: 'NoneType' object has no attribute 'settimeout'
```

The error cannot be reproduced with version before 1.0.0 (0.59.0) or after 1.0.0 (1.0.1)

**Comment by wildraid** [comment]
Wow. `websocket-client` was abandoned for a while. What a surprise. I'll take a look.

I guess, I'll have to add an exclusion for this version in in dependencies.

**Comment by wildraid** [comment]
Ok, it is definitely related to the fix in 1.0.1: https://github.com/websocket-client/websocket-client/commit/0e4298a7b1876f3f8ad97b1ab27f3b283d6ea148

So the exception was always there, but it is muted prior or after 1.0.0.

I've added 1.0.0 to explicit exclusions, it won't hurt: https://github.com/badoo/pyexasol/commit/292e2d9c2da0db953317a221d26a99025a11b8bd

It will go live with the next update.

Thank you.

**Comment by smaspe** [comment]
Cool, thanks!
